### PR TITLE
Script to convert a base64 file from a list a into binary file and upload to the WarRoom

### PR DESCRIPTION
**Pull request by vik-patel**
Script to convert a base64 file from a list a into binary file and upload to the WarRoom

### CODE REVIEW

**Comment by aviadl** [comment]
@vik-patel isnt it this script `b64FromList` which is already in content?

**Comment by vik-patel** [comment]
@aviadl I don't see b64FromList in content

**Comment by aviadl** [comment]
You are right, sorry
I had it only locally, never published it or anything ...

**Comment by vik-patel** [comment]
@aviadl ... ok, where do I upload the test playbook?

**Comment by vik-patel** [comment]
@aviadl I added test playbook, but I did not find an automation to create a list ... I uses "setList" but will error if list does not exist

**Comment by Itay4** [comment]
@vik-patel What is the status of this?

**Comment by vik-patel** [comment]
@Itay4 ... fixed, please check and approve

File: Scripts/script-Base64ListToFile.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  # convert base64 file to binary file
+  bin_file = base64.decodestring(res[0]['Contents'])
+
+  # output file to warroom
+  demisto.results(fileResult(outfilename,bin_file))
+type: python
+tags:
+- Utility
+comment: Will covert Base64 file in a list to a binary file and upload to warroom
```
Comment (Itay4) [inline Scripts/script-Base64ListToFile.yml]: Please change to: `Converts base64 file in a list to a binary file`

File: Scripts/script-Base64ListToFile.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+enabled: true
+args:
+- name: listname
+  required: true
+  description: Listname of Base64 item (need to be a single file in list)
```
Comment (Itay4) [inline Scripts/script-Base64ListToFile.yml]: `Listname` should be `List name`

File: Scripts/script-Base64ListToFile.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+- name: filename
+  description: Optional Warroom Output Filename (default filename is listname)
```
Comment (Itay4) [inline Scripts/script-Base64ListToFile.yml]: Please change to `default filename is the list name`

File: Scripts/script-Base64ListToFile.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+outputs:
```
Comment (Itay4) [inline Scripts/script-Base64ListToFile.yml]: Please add outputs types and add EntryID to the outputs

**Comment by Itay4** [comment]
@vik-patel What is the status of this?

**Comment by vik-patel** [comment]
@anara123 please check and approve

**Pull request merged**